src/main/java/com/marginallyclever/makelangelo/preview/PreviewPanel.java

```diff
@@ -221,16 +240,35 @@ public void display(GLAutoDrawable glautodrawable) {
 		
 		paintBackground(gl2);
 		paintCamera(gl2);
-		
-		gl2.glLineWidth((float)camera.getZoom());
 
 		for( PreviewListener p : previewListeners ) {
 			gl2.glPushMatrix();
 			p.render(gl2);
 			gl2.glPopMatrix();
 		}
+
+		//paintCursor(gl2);
 	}
-	
+
+	private void paintCursor(GL2 gl2) {
+		gl2.glPushMatrix();
+			Rectangle r = this.getBounds();
```

Comment:
Please adjust the identation

Reply:
I use the indentation to very clearly show every push has a pop.  ....I'll do it.